enterprise/kernel/src/main/java/org/neo4j/kernel/impl/enterprise/lock/forseti/ForsetiClient.java

```diff
@@ -700,14 +714,19 @@ public void stop()
     @Override
     public void close()
     {
-        stop();
+        stateHolder.closeClient();
+        waitForAllClientsToLeave();
+        releaseAllLocks();
+    }
+
+    private void releaseAllLocks()
+    {
         if ( hasLocks )
         {
             releaseAllClientLocks();
             clearWaitList();
             hasLocks = false;
         }
-        clientPool.release( this );
```

Comment:
not sure that removal of this line was planned

Reply:
Right, that should have been added back into `close()`. Good catch.